### ISSUE TITLE
token transfer to multiple receivers

### ISSUE THREAD

**Issue by buhtignew** [slimcoin-project/pacli#108]
I've been testing `token transfer` to multiple receivers.

Thus I've set up my address `mtMGkB2mVxzPNZCWHruRmG1SmsSJBtyCUD` that has 3 PoB tokens as main (`address set -a mtMGkB2mVxzPNZCWHruRmG1SmsSJBtyCUD` command) and then tried the following combinations:

- `token transfer fb93cce7aceb9f7fda228bc0c0c2eca8c56c09c1d846a04bd6a59cae2a895974 [mxTq1JuXLMZjaKA8LkdySk3sMwhRz4FEG2, mwLKewF56R8Rje2tAuQRJzDxS9XC1a1q23, myHPHhnncUaiFQGqN1FK2qFtwcfPE4Bkqc] [0.1, 0.2, 0.3]`, 
- `token transfer fb93cce7aceb9f7fda228bc0c0c2eca8c56c09c1d846a04bd6a59cae2a895974 [fresh_after_refill9, another_fresh, usage_checked] [0.1, 0.2, 0.3]`,  
- `token transfer fb93cce7aceb9f7fda228bc0c0c2eca8c56c09c1d846a04bd6a59cae2a895974 [fresh_after_refill9, another_fresh, usage_checked] [1, 1, 1]`, 
- `token transfer fb93cce7aceb9f7fda228bc0c0c2eca8c56c09c1d846a04bd6a59cae2a895974 ['fresh_after_refill9', 'another_fresh', 'usage_checked'] [1, 1, 1]`, 
- `token transfer fb93cce7aceb9f7fda228bc0c0c2eca8c56c09c1d846a04bd6a59cae2a895974 ["mxTq1JuXLMZjaKA8LkdySk3sMwhRz4FEG2", "mwLKewF56R8Rje2tAuQRJzDxS9XC1a1q23", "myHPHhnncUaiFQGqN1FK2qFtwcfPE4Bkqc"] [0.1, 0.2, 0.3]`,
- `token transfer fb93cce7aceb9f7fda228bc0c0c2eca8c56c09c1d846a04bd6a59cae2a895974 ["mxTq1JuXLMZjaKA8LkdySk3sMwhRz4FEG2", "mwLKewF56R8Rje2tAuQRJzDxS9XC1a1q23", "myHPHhnncUaiFQGqN1FK2qFtwcfPE4Bkqc"] ["0.1", "0.2", "0.3"]`  

and got the following message in each case:
```
Error: The receiver and amount parameters have to be strings/numbers or lists.
```
What am I doing wrong?

**Comment by d5000** [comment]
There were two issues here. First there was a bug in the code, which is now fixed as of commit de3910f.

The second issue has to do with the way the shell works. If you enter the lists of receivers and amounts with spaces between the values, they will not be recognized properly, as long as you don't put quotation marks before and after the whole list. You can also simply remove the spaces.

I'm aware that in the help that looked different, I've changed that now.

**Comment by buhtignew** [comment]
The issue is solved, I'm closing it.
